static/js/util/ProfileTab.js

```diff
@@ -56,7 +56,9 @@ class ProfileTab extends React.Component {
   }
 
   saveAndContinue = () => {
-    saveAndContinue.call(this, this.constructor.prototype.validation).then(() => {
+    // this tab is the last tab if it redirects to the dashboard
+    let lastTab = this.nextUrl === "/dashboard";
+    saveAndContinue.call(this, this.constructor.prototype.validation, lastTab).then(() => {
```

Comment:
this is slick! what do you think about deleting the comment? the line (`let lastTab = ...`) is self-explanatory to me.